product_warranty/views/product_warranty.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- return instructions tree view -->
+    <record model="ir.ui.view" id="product_return_instructions_tree_view">
+        <field name="name">product.return.instructions.tree</field>
+        <field name="model">return.instruction</field>
+        <field name="arch" type="xml">
+            <tree string="Return instructions">
+                <field name="name"/>
+                <field name="instructions"/>
+                <field name="is_default"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- return instructions form view -->
+    <record model="ir.ui.view" id="product_return_instructions_form_view">
+        <field name="name">product.return.instructions.form</field>
+        <field name="model">return.instruction</field>
+        <field name="arch" type="xml">
+            <form string="Return instructions">
+                <sheet>
+                    <group name="return_instructions" col='4'>
+                        <field name="name"/>
+                        <field name="is_default"/>
+                    </group>
+                    <separator string="Instructions" colspan="4"/>
+                    <field name="instructions" nolabel="1" colspan="4"/>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="product_return_instructions_action"
+            model="ir.actions.act_window">
+        <field name="name">Products Return Instructions</field>
+        <field name="res_model">return.instruction</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="product_return_instructions_tree_view"/>
+    </record>
+
+    <menuitem action="product_return_instructions_action"
+              groups="base.group_no_one"
+              id="menu_product_return_instructions_action"
+              parent="sale.prod_config_main" sequence="3"/>
+
+    <!-- supplier info tree view -->
+    <record model="ir.ui.view" id="product_supplierinfo_warranty_tree_view">
+        <field name="name">product.supplierinfo.warranty.tree</field>
+        <field name="model">product.supplierinfo</field>
+        <field name="inherit_id" ref="product.product_supplierinfo_tree_view"/>
+        <field name="arch" type="xml">
+            <field name="min_qty" position="before">
+                <field name="warranty_duration"/>
+                <field name="warranty_return_partner"/>
+                <field name="warranty_return_address"/>
+                <field name="active_supplier"/>
+                <field name="return_instructions"/>
+            </field>
+        </field>
+    </record>
+
+    <!-- supplier info form view -->
+    <record model="ir.ui.view" id="product_supplierinfo_warranty_form_view">
```

Comment:
just aesthetic change:
id before model

id pattern:
For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...